tests/core/test_projectors.py

```diff
@@ -287,7 +287,19 @@ class projected_family_level_variable(Variable):
         def formula(family, period):
             return family.household("household_level_variable", period)
 
-    system.add_variables(household_level_variable, projected_family_level_variable)
+    class decoded_projected_family_level_variable(Variable):
```

Comment:
I wonder why the class is named like this, when Python class naming conventions is to use [PascalCase](https://namingconvention.org/python/class-naming.html). But then again, I'm not familiar enough with the code of this project so I'm sure there is a perfectly valid reason for this.